fast64_internal/f3d/f3d_parser.py

```diff
@@ -1309,7 +1309,7 @@ def processCommands(self, dlData, dlName, dlCommands):
 				self.addTriangle(command.params[0:3], dlData)
 			elif command.name == 'gsSP2Triangles':
 				self.addTriangle(command.params[0:3] + command.params[4:7], dlData)
-			elif command.name == 'gsSPDisplayList' or command.name[:10] == 'gsSPBranch':
+			elif (command.name == 'gsSPDisplayList' or command.name[:10] == 'gsSPBranch') and not ignoreCalls:
```

Comment:
You are right, I did this because in some scenes it was importing the parent objects and then also the child objects, so there were duplicates. Instead of doing a full hierarchy, I'll try to have it take the current list of DLs and then exclude any DLs which are called by another DL in the same file.